kbs/docs/kbs_attestation_protocol.md

```diff
@@ -227,6 +229,12 @@ Encryption algorithm used to encrypt the output of the KBS service API.
 The output of the KBS service API. It must be encrypted with the KBS-generated
 ephemeral key.
 
+- `aad` (Required if AEAD is used)
```

Comment:
Hi @tylerfanelli I added the `aad` and `tag` parts to this document. They all follow [RFC7516](https://datatracker.ietf.org/doc/html/rfc7516#appendix-A.2.3)